src/strongbox/http.clj

```diff
@@ -194,12 +195,12 @@
 
                                              ;; count bytes transferred so we can update the job progress (if one exists). taken from:
                                              ;; - https://github.com/dakrone/clj-http/blob/7aa6d02ad83dff9af6217f39e517cde2ded73a25/examples/progress_download.clj
-                                             (let [length (Integer/valueOf (get-in resp [:headers "content-length"] 0))
+                                             (let [length (-> resp (get-in [:headers "content-length"] 0) utils/to-int)
                                                    buffer-size (* 1024 10)]
-                                               (with-open [input (:body resp)
+                                               (with-open [^java.io.InputStream input (:body resp)
```

Comment:
double check this

Reply:
looks like inputstreams are preferred internally in clj-http, think we're safe here.